lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx

```diff
@@ -65,111 +66,119 @@ inline bool varianceComparison(const std::pair<UnsignedInteger, Scalar> &a, cons
   return a.second > b.second;
 }
 
+/* String formatter with blank space right padding */
+String FunctionalChaosSobolIndices::padString(const String & string) const
+{
+  const String intermediateString = OSS() << string;
+  const UnsignedInteger columnWidth = ResourceMap::GetAsUnsignedInteger("FunctionalChaosSobolIndices-PrintColumnWidth");
+  const UnsignedInteger rightPadding = intermediateString.size() < columnWidth ? columnWidth - intermediateString.size() : 0;
```

Comment:
Don't assume that String is an alias to std::str, use `getSize()` instead of `size()`

Reply:
Ok, but this does not build then:
```
FunctionalChaosResult.cxx:116:47: error: ‘const String’ {aka ‘const class std::__cxx11::basic_string<char>’} has no member named ‘getSize’
```

Reply:
`size()` is OK actually.